usagov/usagov-2021
# USAGOV-1738-Life-Events-All-Topics-Card: Added Icon for Life Events

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1738

## Description

Added the svg icon for Life Events All Topics Card

## Type of Changes

  - Add screenshot showing what it should look like


## Testing Instructions



### Change Requirements



### Validation Steps

- On /life-events go to edit
- On the edit page (http://localhost/node/753/edit) select 'Promoted to front page' under Promotion options on the right menu
- In the CSS icon field (under the navigation banner) add: icon-life-events
- Save and go back to the homepage
- Life Events should now have a card in the All Topics section with the icon showing
![Screenshot 2024-08-06 at 9 18 32 AM](https://github.com/user-attachments/assets/67ea7778-4ffa-42c9-b568-e212423b7ae4)

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
